templates/login/login.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}{% load static %}{%block content%}
+
+<h2>Log In</h2>
+<form method="post">
```

Comment:
It will be nice if you can add a bit of style to the login, maybe put the form and the button in the middle of the screen.
take some ideas from the internet on how to make it look nicer
for example:  https://bootsnipp.com/snippets/dldxB

Reply:
> It will be nice if you can add a bit of style to the login, maybe put the form and the button in the middle of the screen. take some ideas from the internet on how to make it look nicer for example: https://bootsnipp.com/snippets/dldxB

I have found this on the internet: https://getbootstrap.com/docs/4.0/getting-started/introduction/#starter-template
but we already have a base html file..
So I didn't want to change it to everybody, When I messed a bit with it I saw it changed on other apps as well (of course because it inherits from the base.html)

I thought maybe inheriting from a different one, but it is code duplication..

I searched online about inheriting from inherited html file (double inheritance, like "grandfather")
but I didn't find anything useful

Do you know of something like that?